assets/src/components/drawerTab.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+
+import { collapseIcon, expandIcon } from "../helpers/icon"
+
+const DrawerTab = ({
```

Comment:
I know this is just based on the existing `Tab` component that existed in another module, but it might be good to write a quick test or two for it now that it's in its own module.

Reply:
sounds like a good idea